offchain/advance-runner/Cargo.toml

```diff
@@ -16,13 +16,16 @@ rollups-events = { path = "../rollups-events" }
 async-trait.workspace = true
 backoff = { workspace = true, features = ["tokio"] }
 clap = { workspace = true, features = ["derive", "env"] }
+contracts = { path = "../contracts" }
```

Comment:
This should be with the other local dependencies (grpc-interfaces, http-healthcheck, rollups-events).